policyengine_us/parameters/gov/states/sc/tax/income/use_tax/group_two.yaml

```diff
@@ -0,0 +1,11 @@
+description: South Carolina levies a local use tax of group 2 counties at this percent of federal adjusted gross income.
+values:
+  2021-05-01: 0.02
+
+metadata:
+  period: year
+  unit: /1
+  label: South Carolina group two county use tax rate
+  reference:
+  - title: South Carolina Sales and Use Tax Manual
+    href: https://dor.sc.gov/resources-site/lawandpolicy/Documents/SC%20Sales%20Tax%20Manual.pdf
```

Comment:
could you add pages to the references which mention the specific rate 

all yaml files